database/seeders/modules/beginning-php.php

```diff
@@ -0,0 +1,123 @@
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+        ],
+        'en' => [
+            [
+                'name' => 'Laracasts Video (Variables)',
+                'url' => 'https://laracasts.com/series/php-for-beginners/episodes/3',
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+            [
+                'name' => 'Codecademy - Variables',
+                'url' => 'https://www.codecademy.com/learn/learn-php/modules/learn-php-variables',
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+            [
+                'name' => 'Laracasts Video (Arrays)',
+                'url' => 'https://laracasts.com/series/php-for-beginners/episodes/6',
+                'type' => Resource::VIDEO_TYPE,
+            ],
+            [
+                'name' => 'Codecademy - Arrays',
+                'url' => 'https://www.codecademy.com/learn/learn-php/modules/learn-php-arrays',
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+            [
+                'name' => 'Laracasts Video - Functions',
+                'url' => 'https://laracasts.com/series/php-for-beginners/episodes/10',
+                'type' => Resource::VIDEO_TYPE,
+            ],
+            [
+                'name' => 'Codecademy - Functions',
+                'url' => 'https://www.codecademy.com/learn/learn-php/modules/introduction-to-functions-in-php',
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+            [
+                'name' => 'Laracasts Video (Classes)',
+                'url' => 'https://laracasts.com/series/php-for-beginners/episodes/12',
+                'type' => Resource::VIDEO_TYPE,
+            ],
+            [
+                'name' => 'Codecademy - Conditionals',
+                'url' => 'https://www.codecademy.com/learn/learn-php/modules/conditionals-logic-php',
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+            [
+                'name' => 'Codecademy - Loops',
+                'url' => 'https://www.codecademy.com/learn/learn-php/modules/php-loops',
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+            [
+                'name' => 'Open Source Learning for PHP (course)',
+                'url' => 'https://www.phpschool.io/',
+                'type' => Resource::COURSE_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Intro to Composer, with Jordi Boggiano',
+                'url' => 'https://laravelpodcast.com/episodes/intro-to-composer-with-jordi-boggiano',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Packages, with Freek Van der Herten & Marcel Pociot',
+                'url' => 'https://laravelpodcast.com/episodes/packages-with-freek-van-der-herten-marcel-pociot',
+                'type' => Resource::AUDIO_TYPE,
+            ],
```

Comment:
Proposal: This should be a part of a new intermediate module surrounding Package Creation
```suggestion
```